io.github.lainsce.Emulsion.json

```diff
@@ -0,0 +1,88 @@
+{
+    "app-id" : "io.github.lainsce.Emulsion",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "40",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "io.github.lainsce.Emulsion",
+    "rename-icon" : "io.github.lainsce.Emulsion",
+    "finish-args" : [
+        "--filesystem=xdg-download",
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--socket=session-bus",
```

Comment:
Why does it need access to session bus?

Reply:
When I removed this, it complained during my builds on Builder, and wouldn't even open the app.

Reply:
You're probably using the wrong app-id somewhere.

Reply:
Doublechecked it, I think it was just me using the Devel in the app-id in my Builder builds. It runs without this, so fixed.